lib/chef/azure/commands/enable.rb

```diff
@@ -66,9 +66,7 @@ def enable_chef
     begin
       configure_chef_only_once
 
-      install_chef_service if @exit_code == 0
-
-      enable_chef_service if @exit_code == 0
+      enable_chef_service

```

Comment:
was dropping `if @exit_code == 0` intentional? what if the settings file is missing?